src/main/webapp/app/entities/flair-bi/scheduler/scheduler-dialog.controller.js

```diff
@@ -30,6 +30,11 @@
         vm.schedulerData={};
         vm.added = added;
         vm.removed = removed;
+        vm.WebhookList=[];
```

Comment:
.no need to declare new array vm.WebhookList for webhooks.    assign_report.communication_list can we used

 "assign_report": {
                "channel": [],
                "communication_list":{"email":[],"teams":[]}
            },

Reply:
okay